integrationtests/mayavi/test_contour.py

```diff
@@ -177,6 +177,68 @@ def do(self):
 
         # If we have come this far, we are golden!
 
+    def do_profile(self):
```

Comment:
This code does not check the specifics changes that are put forward by the this PR.